nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,118 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
```

Comment:
Maybe:

> It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.

would be better as:

Please note that the  SQL Server transport does **not** use or depend on SQL Server Service Broker.